### DISCUSSION TITLE
电脑搭建alist，手机nplayer无法访问，报错timeout was reached

### DISCUSSION THREAD

**Discussion by 0jiuxian0** [AlistGo/alist#2261] [Q&A] [answered]
我在电脑上搭建了alist，并且用raidrive挂载到本地盘，访问视频图片没有问题。之后想用手机nplayer访问alist，结果一直报错timeout was reached(28)。测试过电脑ping手机地址是可以ping通的。

电脑系统：win10 20H2
alist版本：3.3.0
手机系统：MIUI12.5.2
nplayer版本：1.7.7.7_191219

webdav配置如下
![image](https://user-images.githubusercontent.com/71592475/200562273-bd207aa8-c5ed-4117-9cef-609cf203ce76.png)
![image](https://user-images.githubusercontent.com/71592475/200562187-04896e13-5c22-4451-b7c3-cc4388bec8b7.png)

**Comment by chisora02** [comment]
路径前加一个/试试

**Reply by 0jiuxian0** [reply]
试过了，还是一样的结果

**Reply by chisora02** [reply]
我之前想用pojobox挂webdrive的，可能是app本身的问题，他设置不全我怎么都挂不上，但是infuse可以，但是infuse访问不了其他文件，然后我就直接用浏览器访问的alist，地址http://+ip地址+：端口号，你试试直接访问，但是得是同一个网络

**Reply by 0jiuxian0** [reply]
我一开始电脑搭建好后，就尝试用手机浏览器去访问，但是也是连接超时。后面我才尝试使用其他的软件进行访问，有试过solid explorer，都连接不上。我觉得是哪里设置有问题，电脑里高级共享设置里的选项我都开启了，电脑ping手机也能ping通，就不知道哪里还有问题。我刚才尝试手机开热点来测试，还是同样的结果。
![image](https://user-images.githubusercontent.com/71592475/200720702-9abd44a2-cc06-4810-bea0-4b57411f4f41.png)

**Reply by chisora02** [reply]
我也不太明白了，会不会是网络的问题，我’用的是a类网

**Reply by Faxmell** [reply]
https那个选项关了
另外用户选你有管理权限的那个账户
这样试下
还不行我就不知道了

**Comment by anwen-anyi** [comment]
我也遇到了类似的(和你的不一样）
 - 我在本地搭建的Alist和服务器搭建的Alist
 - 本地搭建的和服务器搭建的在安卓nplayer就能挂载没任何问题
 - 然后呢我把本地搭建的和服务器搭建的挂载到IOS，服务器的没事儿能挂载，本地搭建的就 提示：**Couldn't connect to server**
 
有点儿迷惑.. 哈哈

不过在手机访问本地搭建的内网 IP:5244 是能访问的..

**Comment by wscqstu** [comment]
你用webdav连接，地址应该是：http://192.168.0.125:5244/dav呀，这个会弹出对话框让你输入用户名和密码，
这个用户名和密码，我输入系统地用户名和密码就是错误，我不知道这个用户名和密码应该输入什么

**Comment by chisora02** [comment]
不是输入alist的用户名和密码吗

**Comment by makergit365** [comment]
把https选项关了啊，去查一下https的区别，不想搞明白就以后干啥都全别开，除非看到是https

**Comment by 0jiuxian0** [comment, marked as answer]
alist更新到3.6.0，这个问题就解决了，具体原因不明，应该是旧版本的bug。

**Reply by xinxinenjoy** [reply]
我的还是无法解决，电视上的KODI访问不了

**Reply by q1azq1a** [reply]
端口改为80

**Comment by omgblack7** [comment]
dav前面要添加/
